src/app/globals.css

```diff
@@ -54,6 +54,10 @@ button {
   padding: 0.5em 1em;
 }
 
+button:hover {
+  cursor: pointer;
+}
```

Comment:
If we like to have the pointer on every button, it's more common to set it in every case (not only hover).

```suggestion
button {
  cursor: pointer;
}
```

Reply:
solved